packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -82,6 +80,8 @@ export const disableDatastoreSweepKey = "Fluid.GarbageCollection.DisableDataStor
 export const detectOutboundRoutesViaDDSKey = "Fluid.GarbageCollection.DetectOutboundRoutesViaDDS";
 /** Config key to disable auto-recovery mechanism that protects Tombstones that are loaded from being swept (use true) */
 export const disableAutoRecoveryKey = "Fluid.GarbageCollection.DisableAutoRecovery";
+/** Config key to turn GC on / off for testing. */
+export const runGCTestKey = "Fluid.GarbageCollection.Test.RunGC";
```

Comment:
Maybe we should remove this too? Don't think we need to test disabling GC.

Reply:
I say leave it as an escape valve in case there's some scenario where we or a customer hits something and wants to disable GC as a workaround.